intelmq/bots/parsers/n6/parser_n6stomp.py

```diff
@@ -41,10 +41,10 @@
                              "identifier": "malware configuration"}
 mapping['phish']        = {"taxonomy": "Fraud",
                            "type": "phishing", "identifier": "phishing"}
-mapping['proxy']        = {"taxonomy": "Vulnerable",
-                           "type": "proxy", "identifier": "open proxy"}
-mapping['sandbox-url']  = {"taxonomy": "ignore",
-                           "type": "ignore", "identifier": "ignore me"}
+mapping['proxy']        = {"taxonomy": "Other",
```

Comment:
might be replaced in the future by the taxonomy "accessible". But that's the future....